exp/controllers/azuremanagedmachinepool_reconciler.go

```diff
@@ -23,15 +23,23 @@ import (
 
 	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2020-06-30/compute"
 	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/cluster-api-provider-azure/azure"
 	"sigs.k8s.io/cluster-api-provider-azure/azure/scope"
 	"sigs.k8s.io/cluster-api-provider-azure/azure/services/agentpools"
 	"sigs.k8s.io/cluster-api-provider-azure/azure/services/scalesets"
 	"sigs.k8s.io/cluster-api-provider-azure/util/tele"
+	"sigs.k8s.io/cluster-api/controllers/remote"
 
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	// ManagedMachinePoolScopeName is the sourceName, or more specifically the UserAgent, of client used in AMMP reconciliation.
+	ManagedMachinePoolScopeName = "azuremanagedmachinepoolmachine-scope"
```

Comment:
self-nit: probably should be private or somewhere else for reuse (like under scope package as a variable)